ignite/cmd/plugin.go

```diff
@@ -366,18 +372,19 @@ func NewPluginUpdate() *cobra.Command {
 
 func NewPluginAdd() *cobra.Command {
 	cmdPluginAdd := &cobra.Command{
-		Use:   "add [path] [args]",
+		Use:   "add [path] [key:value]...",
```

Comment:
Is there a reason to use ":" as separator?

```suggestion
		Use:   "add [path] [key=value]...",
```

Using "=" would be more intuitive and potentially make malformed argument errors clearer.

Reply:
done [e9aa1af](https://github.com/ignite/cli/pull/3259/commits/e9aa1af13541dd4045d42445619e4eff1f4a3177)